src/Whoops/Resources/views/frame_code.html.php

```diff
@@ -29,20 +29,6 @@
             $start = key($range) + 1;
             $code  = join("\n", $range);
         ?>
-            <style>
-             #frame-code-0 .linenums li:nth-child(<?= $line-$start ?>) {
-                 background-color: rgba(255, 100, 100, .07); 
```

Comment:
The background-color is defined in [whoops.base.css](https://github.com/jonasdt/whoops/blob/915a1ad1762e443280df6f42e35033a0aaa3bea1/src/Whoops/Resources/css/whoops.base.css#L353-L358).